packages/login/network/settings.js

```diff
@@ -103,6 +103,24 @@ export const MIDDLEWARE_MAP = {
     mainnet: 'https://middleware-ethereum-mainnet-rest.chronobank.io/',
     testnet: 'https://middleware-ethereum-testnet-rest.chronobank.io/',
   },
+  txHistory: {
```

Comment:
Proxy all calls to the NemNode/EthereumNode/BitcoinNode
They all have configured axios instances.